README.md

```diff
@@ -374,32 +374,34 @@ for await (let b of resp) {
 ```
 note: to get the date of response samples you could do this `console.log(new Date(resp['AAPL'][0].startEpochTime*1000))`
 
-#### Last trade
+#### Get the Latest trade
 
 ```ts
-lastTrade(
-  symbol: string)
-) => Promise<LastTradeObject>
+getLatestTrade(
+  symbol: string,
+  config: any
+) => Promise<AlpacaTrade>
 ```
 ###### example
 ```js
-this.alpaca.lastTrade('AAPL').then((response) => {
-          console.log(response)
-        })
+const trade = await this.alpaca.getLatestTrade('AAPL');
```

Comment:
Using the `this` keyword isn't necessary, I would remove then from the whole readme's examples.